analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd

```diff
@@ -122,7 +122,7 @@ add_xy_df <- add_ploidy_df %>%
     germline_sex_estimate == "Male" & copy.num == (0.5 * tumor_ploidy) ~ "neutral",
     # there are some instances where there are chromosome Y segments are in
     # samples where the germline_sex_estimate is Female
-    chrom == "chrY" & germline_sex_estimate == "Female" & copy.num > 0 ~ "unknown",
+    chrom == "chrY" & germline_sex_estimate == "Female" & copy.num > 0 ~ as.character(NA),
```

Comment:
Can you revert this for this PR, as it is the main code change in #182

Reply:
@jharenza the branch in [current PR](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/181) is now merged into [#182](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/182)\\

We can ignore this PR and merge only the other one